apps/web/src/pages/Lesson.tsx

```diff
@@ -398,8 +398,8 @@ const Lesson: React.FC = () => {
             startAt: lesson.data.lesson.start,
           }}
           leave={() => {
-            session.leave();
-            sessionManager.leave();
+            session.leave(); // this shall close/stop related user media (video, share screen, etc)
+            sessionManager.leave(); // this shall close the connection (websocket), and invoke db mutations.
             if (lessonMembers.current.role === IUser.Role.Student) {
               const query = asRateLessonQuery({
```

Comment:
We can extend this to also include the lesson start in the url 

https://github.com/litespace-org/litespace/blob/dd8d17ccfc3b17546f72518e72133a275afb8807/apps/web/src/lib/query.ts#L7-L15